src/main/scala/fs2/async/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
+
+import fs2.util.Catchable
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+package object async {
+
+
+  /**
+   * Create a new continuous signal which may be controlled asynchronously.
+   * Note that this would block any resulting processes (discrete, continuous) until any signal value is set.
+   */
+  def signalUnset[F[_]:AsyncExt,A]: Stream[F,mutable.Signal[F,A]] =
```

Comment:
Ok to me this was just  a syntax sugar to `Signal[Option[A]].pipe(collect{case Some(a) => a})` put perhaps with pipe on signal now this has no justification.